packages/garbo/src/familiar/barfFamiliar.ts

```diff
@@ -235,7 +245,14 @@ export function barfFamiliar(): Familiar {
     HIGHLIGHT,
   );
 
-  return best.familiar;
+  const jellyfish = fullMenu.find(
+    ({ familiar }) => familiar === $familiar`Space Jellyfish`,
+  );
+
+  return {
+    familiar: best.familiar,
+    extraValue: extraValue(best, meatFamiliarEntry, jellyfish),
```

Comment:
I think I need some documentation about what `extraValue` means here. extra compared to what?